packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -268,12 +269,12 @@ const VirtualListBaseFactory = (type) => {
 				// remove a function for preventing native scrolling by Spotlight
 				if (containerNode && containerNode.removeEventListener) {
 					containerNode.removeEventListener('scroll', this.preventScroll);
-					containerNode.removeEventListener('keydown', this.onKeyDown);
 				}
 			}
 
 			if (containerNode && containerNode.removeEventListener) {
 				containerNode.removeEventListener('keydown', this.onKeyDown);
+				containerNode.removeEventListener('keyup', this.revertSpotlightState);
```

Comment:
Hi @rundmt 
I could not find where `revertSpotlightState` function is. Am I missing something?

Reply:
`revertSpotlightState` might be `resumeSpotlight`?